doc/sources/quick-start.rst

```diff
@@ -14,17 +14,455 @@
+     - Python 3.10
+     - Python 3.11
+     - Python 3.12
+   * - Linux* OS
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+   * - Windows* OS
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+
+
+
+Install from Anaconda* Cloud
+********************************************
+
+To prevent version conflicts, we recommend installing `scikit-learn-intelex` into a new conda environment.
+
+.. tabs::
+
+   .. tab:: Conda-Forge channel
+
+      Recommended by default. 
+      
+      To install, run::
+
+        conda install scikit-learn-intelex -c conda-forge
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+         * - Windows* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+
+
+   .. tab:: Intel channel
+
+      Recommended for the Intel® Distribution for Python users. 
+
+      To install, run::
+
+        conda install scikit-learn-intelex -c intel
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+         * - Windows* OS
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+           - [CPU, GPU]
+ 
+
+
+   .. tab:: Main channel
+
+      To install, run::
+
+        conda install scikit-learn-intelex
+      
+      .. list-table:: **Supported Configurations**
+         :header-rows: 1
+         :align: left
+
+         * - OS / Python version
+           - Python 3.8
+           - Python 3.9
+           - Python 3.10
+           - Python 3.11
+           - Python 3.12
+         * - Linux* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+         * - Windows* OS
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+           - [CPU]
+
+
+
+Build from Sources
+**********************
+
+You can build Intel® Extension for Scikit-learn* from  sources in the `Intel(R) Extension for Scikit-learn* repository <https://github.com/intel/scikit-learn-intelex>`_.
+
+
+Follow the steps below to build and install the completed package:
+
+1. Check prerequisites:
+
+   * `daal4py <https://github.com/intel/scikit-learn-intelex/blob/master/daal4py/INSTALL.md>`_ >= 2021.4
```

Comment:
mmm - this should be rephrased so you would need same daal4py version as version of sklearnex your are planning to build

Reply:
fixed